src/components/partner/dto/create-partner.dto.ts

```diff
@@ -36,9 +36,15 @@ export abstract class CreatePartner {
   })
   readonly pmcEntityCode?: string;
 
-  @Field({ nullable: true })
+  @Field({
+    nullable: true,
+    deprecationReason: 'Use `growthPartnersClient` instead',
+  })
```

Comment:
Maintaining this _input_ field (as deprecated) is appropriate for a graceful migration. But they need to be used. They should be mapped to the new input properties in the service/repo.